lib/tasks/event.rake

```diff
@@ -1,7 +1,7 @@
 namespace :event do
   desc "Create index for events"
   task :create_index => :environment do
-    Event.__elasticsearch__.create_index!
+    Event.__elasticsearch__.create_index!(index: ENV['INDEX_NAME']||=nil)
```

Comment:
Can you explain this line?

Reply:
I just wanted to have a handy option to generate an index with any given name if needed.

For the events index change, I wanted to do the change in two steps.

1. Create a secondary index. (This will help me to do that)
2. Do the reindex.

Reply:
I understand. But how do you remember the index name? And how does a team colleague know that name?

And what does `||= nil` do?